examples/01_simple_agent_example/simple_agent_example.py

```diff
@@ -20,9 +21,11 @@ async def start_agent() -> None:
         processors=[],  # processors can fetch extra data, check images/audio data or transform video
         # llm with tts & stt. if you use a realtime (sts capable) llm the tts, stt and vad aren't needed
         llm=llm,
-        tts=cartesia.TTS(),
+        tts=elevenlabs.TTS(),
         stt=deepgram.STT(),
         turn_detection=vogent.TurnDetection(),
+        profiler=Profiler(),
```

Comment:
Let's disable profiler by defaut
```suggestion
        # Uncomment this line to enable profiling.
        # profiler=Profiler(),
```